regulations/urls.py

```diff
@@ -92,11 +93,7 @@
     url(rf'^(?P<label_id>{match_paragraph})/CURRENT$',
         redirect_by_current_date, name='redirect_by_current_date'),
 
-    # A regulation section with chrome
-    # Example: http://.../201-4/2013-10704
-    url(rf'^(?P<label_id>{match_section})/(?P<version>{match_version})$',
-        SectionView.as_view(),
-        name='chrome_section_view'),
+    path('<part>/<section>/<version>/', SectionView.as_view(), name='reader_view'),
```

Comment:
locking this down by type would make sense in the future. Anything we can do to make route's explicit will make it less likely for the wrong route to be triggered. Otherwise (and currently) order matters.